tests/scan/test_performance_bias_detector.py

```diff
@@ -92,67 +92,102 @@ def test_selects_issues_with_benjamini_hochberg(titanic_model, titanic_dataset):
     issues = detector.run(titanic_model, titanic_dataset, features=["Name", "Sex", "Pclass"])
     assert len(issues) == 3
 
-    detector = PerformanceBiasDetector(alpha=1e-10)
+    detector = PerformanceBiasDetector(alpha=1e-15)
 
     issues = detector.run(titanic_model, titanic_dataset, features=["Name", "Sex", "Pclass"])
     assert len(issues) == 2
 
 
 def test_calculate_slice_metrics():
+    SLICE_SIZE = 500
+    np.random.seed(42)
```

Comment:
should have the seed somewhere in the detector, not setting the global seed here

Reply:
Yes, not sure why I put it there